File: A. I Wanna Be the Guy/solve.py
n = int(input())
a = list(map(int, input().split()))
b = list(map(int, input().split()))

x,a = a[0],a[1:]
y,b = b[0],b[1:]




c = [0] * n
for i in a:
	c[i-1] = 1
for i in b:
	c[i-1] = 1

win = True
for i in c:
	if i != 1:
		win = False
		break

if win:
	print("I become the guy.")
else:
	print("Oh, my keyboard!")

File: A. I Wanna Be the Guy/solve.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
	
	int n;
	cin >> n;

	vector<int> a(n,0);

	int x;
	cin >> x;
	for (int i = 0; i < x; ++i)
	{
		int v;
		cin >> v;
		a[v-1] = 1;
	}

	int y;
	cin >> y;
	for (int i = 0; i < y; ++i)
	{
		int v;
		cin >> v;
		a[v-1] = 1;
	}


	bool win = true;
	for (int i = 0; i < n; ++i)
	{
		if (a[i] != 1){
			win = false;
			break;
		}
	}

	if (win){
		cout << "I become the guy." << endl;
	}
	else{
		cout << "Oh, my keyboard!" << endl;
	}
	
	

	return 0;
}